src/i18n/locales/ja.json

```diff
@@ -0,0 +1,99 @@
+{
+  "blog.title": "Node.js ブログ",
+  "blog.description": "Node.js の最新ニュース、事例、チュートリアル、リソースなどをご紹介しています。",
+  "blog.authors.list.title.by": "著者",
+  "blog.authors.list.title.and": "and",
+  "components.downloadAdditional.title": "他のダウンロード",
+  "components.downloadHeader.navigation.activeSection": "ダウンロード",
+  "components.downloadHeader.navigation.title": "ダウンロード",
+  "components.downloadHeader.navigation.nodeVersion": "{lts, select, true {LATEST LTS}other {CURRENT}} VERSION {nodeVersion}",
```

Comment:
this line can be translated look up ad French transaltion if you want an example.
if you want help ping me